acme/acme/magic_typing.py

```diff
@@ -1,12 +1,17 @@
 """Shim class to not have to depend on typing module in prod."""
 import sys
+import warnings
+
+warnings.warn("Module acme.magic_typing is deprecated and will be removed in a future release.",
```

Comment:
```suggestion
warnings.warn("acme.magic_typing is deprecated and will be removed in a future release.",
```
Alternatively we could say, "The module acme.magic_typing" or "The acme.magic_typing module", but I think the phrasing is a bit weird when using module without "the".